example_ws/src/example_pkg/test/sample_rostest.test

```diff
@@ -1,10 +1,14 @@
 <launch>
 
-    <env name="COVERAGE_PROCESS_START" value=".coveragerc"/>
+    <arg name="coverage" default="false"/>
```

Comment:
During tests, we want this to default to true, right?
We don't necessarily need to upload the results, but my workflow (at least in a past job that cared about coverage) was to generate the coverage report, examine it for gaps and then write more tests.

Reply:
if ENABLE_COVERAGE_TESTING is set, this gets set true. That'd be the "standard" way of doing it (so it isn't always generating coverage). However, you could certainly set this to tru if you wanted - but note that it gets overwritten by the cmake (setting it equal to ENABLE_COVERAGE_TESTING)